pkg/kubelet/volumemanager/reconciler/reconciler_test.go

```diff
@@ -1138,17 +1138,18 @@ func Test_Run_Positive_VolumeFSResizeControllerAttachEnabled(t *testing.T) {
 			// Start the reconciler to fill ASW.
 			stopChan, stoppedChan := make(chan struct{}), make(chan struct{})
 			go func() {
+				defer close(stoppedChan)
 				reconciler.Run(stopChan)
-				close(stoppedChan)
 			}()
 			waitForMount(t, fakePlugin, volumeName, asw)
 			// Stop the reconciler.
 			close(stopChan)
 			<-stoppedChan
 
 			// Simulate what DSOWP does
-			pv.Spec.Capacity[v1.ResourceStorage] = tc.newPVSize
-			volumeSpec = &volume.Spec{PersistentVolume: pv}
+			pvWithSize := pv.DeepCopy()
```

Comment:
maybe add a comment briefly explained why this is needed?

Reply:
👌

Reply:
did you add it somewhere?

Reply:
I need to update the commit message later.

The comment was  updated. Would you check if it makes sense？

Reply:
sure, The reason I care about this part is that I want to make sure the tests are doing something represent the real case. For example, we might discover sometimes in the actual code we also need to use deepcopy to avoid race condition. But in this case, only test needs to use deepcopy of object because it represents the real case, I think. Does it make sense?

Reply:
I think I should go through the volume reconciling logics to figure out whether it has a potential data race problem in real cluster.

Reply:
I checked the first fix, I think it is good, but I haven't checked the second one. Thanks!